Automattic/jetpack
# Fixes search results extended view for searches hitting `isMultiSite`

## Proposed changes:

* Removes checks against `is_multisite()` for search results post date.
* Fixes bad ternary meaning that extended search results were not shown to searches identified by `isMultiSite`

### Other information:


## Jetpack product discussion



Slack: /archives/C02ME06LF/p1677248054800969?thread_ts=1677094738.849489&cid=C02ME06LF

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Visit https://wordpress.com/forums?s=test and see that extended details are missing  from search results
* Sandbox the forums, and run `bin/jetpack-downloader test jetpack update/add-date-to-search-results`
* Extended search result details should show as expected.


![Screenshot 2023-02-27 at 15 21 42](https://user-images.githubusercontent.com/3149464/221604483-123d4ba8-e9a5-41df-a3e1-57b91c9f67ce.png)
